Lumberjack.xcodeproj/project.pbxproj

```diff
@@ -1989,6 +1992,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/Frameworks";
 				MODULEMAP_FILE = Framework/Lumberjack/CocoaLumberjack.modulemap;
+				ONLY_ACTIVE_ARCH = NO;
```

Comment:
If you're using carthage, which is one of the recommended install methods, shouldn't this be `NO`, so it can be run on all architectures as it's not built on the fly.